src/outpack/location.py

```diff
@@ -71,6 +82,149 @@ def outpack_location_rename(old, new, root=None, *, locate=True):
     update_config(config, root.path)
 
 
+def location_resolve_valid(
+    location, root, include_local, include_orphan, allow_no_locations
+):
+    if location is None:
+        location = outpack_location_list(root)
+    elif isinstance(location, str):
+        if location not in outpack_location_list(root):
+            msg = f"Unknown location: '{location}'"
+            raise Exception(msg)
+        else:
+            location = [location]
```

Comment:
```suggestion
        location = [location]
```

I feel this is more our usual style.